code/game/objects/structures.dm

```diff
@@ -91,6 +91,8 @@
 		adjusted_climb_time *= 0.25 //aliens are terrifyingly fast
 	if(HAS_TRAIT(user, TRAIT_FREERUNNING)) //do you have any idea how fast I am???
 		adjusted_climb_time *= 0.8
+	if(HAS_TRAIT(user, TRAIT_FAST_CLIMBER))
+		adjusted_climb_time = 7
```

Comment:
These climb speeds should not be overriden like this, it makes it impossible to balance, if you want to make them climb faster just use a low multiplication.

Reply:
i dont think that's true, this augment *should* override freeruning and other bonuses *because* balancing, if we let them stack then it *migh* become a mess.